app/lib/actions/katello/content_view/destroy.rb

```diff
@@ -12,8 +12,10 @@ def plan(content_view, options = {})
 
           sequence do
             concurrence do
-              content_view.content_view_versions.each do |version|
-                plan_action(ContentViewVersion::Destroy, version, options)
+              unless options.fetch(:force_destory_cv, false)
+                content_view.content_view_versions.each do |version|
+                  plan_action(ContentViewVersion::Destroy, version, options)
+                end
```

Comment:
We don't need to run this for force destory cause the versions are deleted as part of Katello::ContentView::Remove where we pass all env and versions in case of force_destroy.